hack/tools/Makefile

```diff
@@ -117,6 +121,11 @@ $(CONVERSION_GEN): go.mod
 	mkdir -p $(BIN_DIR)
 	go build -tags=tools -o $(BIN_DIR) k8s.io/code-generator/cmd/conversion-gen
 
+controller-gen: $(CONTROLLER_GEN) $(SRCS) ## Build controller-gen
+$(CONTROLLER_GEN): go.mod
```

Comment:
This is different from controller-gen in root Makefile. Could we make both consistent?